apps/portal/locale/en.json

```diff
@@ -20,7 +20,10 @@
       },
       "search": {
         "title": "Search Teams & Events",
-        "placeholder": "Search for teams or events..."
+        "placeholder": "Search for teams, events, or locations...",
+        "search-button": "Search",
+        "teams-tab": "Teams",
```

Comment:
I think these changes are outdated, revert changes to this file that are not in the "Events" key